### PR TITLE
Stability tests doc

### PR DESCRIPTION
**Pull request by PrzemyslawBanasiak13**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -252,6 +252,7 @@ nav:
               - 'Device boot measure': unified-test-documentation/dasharo-performance/403-device-boot-measure.md
               - 'CPU temperature measure': unified-test-documentation/dasharo-performance/401-cpu-temperature.md
               - 'CPU frequency measure': unified-test-documentation/dasharo-performance/402-cpu-frequency.md
+              - 'Stability': unified-test-documentation/dasharo-performance/404-stability.md
```
Comment (sulewskiprzemyslaw) [inline mkdocs.yml:255]: @PrzemyslawBanasiak13 These tests are prepared for the specific platform, so please add also a link in the test matrix.

File: docs/unified-test-documentation/dasharo-performance/404-stability.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+**Expected result**
+
+Platform should remain in boot menu without rebooting during whole observation.
+
+### STB001.002 Verify if no reboot occurs in the OS (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that DUT booted to firmware will not reboot during
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-performance/404-stability.md:49]: Firmware? Or maybe Ubuntu?

File: docs/unified-test-documentation/dasharo-performance/404-stability.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Dasharo Performance: Stability
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-performance/404-stability.md:1]:
```suggestion
# Dasharo Performance: Platform stability
```

File: mkdocs.yml (hunk 1 above)
Comment (sulewskiprzemyslaw) [inline mkdocs.yml:255]:
```suggestion
              - 'Platform stability': unified-test-documentation/dasharo-performance/404-stability.md
```

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-performance/404-stability.md (hunk 2 above)
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-performance/404-stability.md]: Fixed

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**